### PR TITLE
🦄 refactor: rm beta

### PR DESCRIPTION
**Pull request by noyyyy**

### CODE REVIEW

File: src/RebornPortalStorage.sol [hunk 1]
```diff
@@ -46,7 +46,8 @@ contract RebornPortalStorage is IRebornDefination {
     // extra reward to parent referrer
     uint256 internal _extraReward;
 
-    bool internal _isBeta;
+    // uesless var for backward compatibility
+    bool internal _g;
```
Comment (yuanjunliang) [inline src/RebornPortalStorage.sol:50]: delete _incarnateCountLimit

File: src/RebornPortal.sol
Comment (yuanjunliang) [inline src/RebornPortal.sol:1]: delete modifier of checkIncarnationCount and limit of incarnate function

File: src/RebornPortalStorage.sol (hunk 1 above)
Comment (noyyyy) [reply src/RebornPortalStorage.sol:50]: it's necessary

File: src/RebornPortal.sol
Comment (noyyyy) [reply src/RebornPortal.sol:1]: it's necessary

**Review by yuanjunliang — APPROVED** [review]

**Pull request merged**